py-polars/polars/expr/binary.py

```diff
@@ -92,6 +190,29 @@ def encode(self, encoding: TransferEncoding) -> Expr:
         -------
         Binary array with values encoded using provided encoding
 
+        Examples
+        --------
+        >>> colors = pl.DataFrame(
+        ...     {
+        ...         "name": ["black", "yellow", "blue"],
+        ...         "code": [b"\x00\x00\x00", b"\xff\xff\x00", b"\x00\x00\xff"],
+        ...     }
+        ... )
+        >>> colors.select(
+        ...     "name",
+        ...     "code",
+        ...     pl.col("code").bin.encode("hex").alias("code_encoded_hex"),
+        ... )
```

Comment:
It's cleaner to use `with_columns` here.

Reply:
Good point, I didn't realize. Updated, thanks for the feedback and the review!